modules/spring/src/test/java/org/apache/ignite/transactions/spring/GridSpringTransactionService.java

```diff
@@ -26,12 +28,16 @@
  * Service.
  */
 public class GridSpringTransactionService {
+    /** */
+    @Autowired
```

Comment:
Seems the service should be annotated with `@Service`

Reply:
I think it is not really necessary since we don't use Spring class auto-detection.